pkg/crc/machine/start.go

```diff
@@ -571,7 +571,7 @@ func updateSSHKeyPair(sshRunner *crcssh.Runner) error {
 	}
 
 	logging.Info("Updating authorized keys ...")
-	cmd := fmt.Sprintf("echo '%s' > /home/core/.ssh/authorized_keys; chmod 644 /home/core/.ssh/authorized_keys", publicKey)
+	cmd := fmt.Sprintf("echo '%s' | sudo tee /home/core/.ssh/authorized_keys;sudo chmod 644 /home/core/.ssh/authorized_keys", publicKey)
```

Comment:
Can we do this `chmod` in `snc` rather than in `crc`? Can you indicate in the commit log that this would fix CI against 4.8 bundles/snc master?